### ISSUE TITLE
Working on Kodi18, but NOT with kodi17

### ISSUE THREAD

**Issue by Animizio** [im85288/service.upnext#5]
Just tried it with KODi 18 and it works.
But with Kodi17 its not working.
Maybe you have a fix or anything solution?

**Comment by im85288** [comment]
Hi thanks for the feedback, for me it worked with Kodi 17. I'm about to do a thread in the kodi forum announcing it. Post a link to your log if it still does not work with version 0.0.6. Eventually though it will be release only in the Leia repo and onwards

**Comment by Animizio** [comment]
Yes its still an issue. Ich tried it a Slot and also my mates, Same result: Not working with 17, nur with 18 it does.

**Comment by im85288** [comment]
No problem, as mentioned above create a new issue following the guidelines above as it provides me with more info on what exactly is going on
